GrowthViz-adults.py

```diff
@@ -237,45 +247,6 @@
 # 
 # This next cell creates an interactive tool that can be used to explore patients. The `sumstats.add_mzscored_to_merged_df` function will add modified Z Scores for height, weight and BMI to `merged_df`. The tool uses [Qgrid](https://github.com/quantopian/qgrid) to create the interactive table. Clicking on a row will create a plot for the individual below the table.
 
-# In[20]:
```

Comment:
Did we remove this chunk from the adults notebook intentionally?

Reply:
It looks like this was removed sometime by mistake - not sure when! Please restore this block of code to the notebook.

Reply:
For easy reference the code that is missing can be found at https://github.com/mitre/GrowthViz/blob/main/GrowthViz-adults.py#L243